moveit_py/src/moveit/moveit_core/collision_detection/collision_matrix.cpp

```diff
@@ -40,16 +40,34 @@ namespace moveit_py
 {
 namespace bind_collision_detection
 {
-std::pair<bool, collision_detection::AllowedCollision::Type>
-getEntry(const std::shared_ptr<collision_detection::AllowedCollisionMatrix>& acm, const std::string& name1,
-         const std::string& name2)
+// TODO: Create a custom typecaster/revise the current implementation to return std::pair<bool,
+// collision_detection::AllowedCollision::Type>
```

Comment:
I like that, can you open an issue for that it is better tracked?

Reply:
https://github.com/ros-planning/moveit2/issues/2553